docs-source/content/userguide/istio/istio.md

```diff
@@ -0,0 +1,275 @@
+method.  When the operator pod starts, you will notice that Istio automatically
+injects an `initContainer` called `istio-init` and the envoy container `istio-proxy`.
+
+You can check this using the following commands:
+
+```
+$ kubectl --namespace weblogic-operator get pods
+$ kubectl --namespace weblogic-operator get pod weblogic-operator-xxx-xxx -o yaml
+```
+
+In the second command, change `weblogic-operator-xxx-xxx` to the name of your pod.
+
+#### Creating a domain with Istio support
+
+You can configure your domains to run with Istio automatic sidecar injection enabled.
+Before creating your domain, create the namespace you wish to run the domain in,
+and label it for automatic injection.
+
+```
+$ kubectl create namespace domain1
+$ kubectl label namespace domain1 istio-injection=enabled
+```
+
+To enable the Istio support for a domain, you need to add the
+`configuration` section to your domain custom resource YAML file as shown in the
+following example:  
+
+```
+apiVersion: "weblogic.oracle/v7"
+kind: Domain
+metadata:
+  name: domain2
+  namespace: domain1
+  labels:
+    weblogic.resourceVersion: domain-v2
+    weblogic.domainUID: domain2
+spec:
+  ... other content ...
+  configuration:
+    istio:
+      enabled: true
+      readinessPort: 8888
+```
+
+To enable the Istio support, you must include the `istio` section
+and you must set `enabled: true` as shown.  The `readniessPort` is optional
+and defaults to `8888` if not provided, it is used for readiness health check.
+
+##### How Istio-enabled domains differ from regular domains
+
+Istio enforces a number of requirements on Pods.  When you enable Istio support in the domain resource, the
+introspector job automatically creates configuration overrides with the necessary channels for the domain to satisfy Istio's requirements, including:
+
+When deploying a domain with Istio sidecar injection enabled.  The WebLogic Server Kubernetes Operator automatically add the following network
+channels via configuration overrides.
+
+https://istio.io/latest/docs/ops/configuration/traffic-management/protocol-selection/
+
+For non-SSL traffic:
+
+|Name|Port|Protocol|Exposed as a container port|
+|----|----|--------|-----|
+|http-probe|From configuration istio readinessPort|http|N|
+|tcp-default|server listening port|t3|Y|
+|http-default|server listening port|http|Y|
+|tcp-snmp|server listening port|snmp|Y|
+|tcp-cbt|server listening port|CLUSTER-BROADCAST|N|
+|tcp-iiop|server listening port|http|N|
+
+For SSL traffic, if SSL is enabled on the server:
+
+|Name|Port|Protocol|Exposed as a container port|
+|----|----|--------|-----|
+|tls-default|server SSL listening port|t3s|Y|
+|https-secure|server SSL listening port|https|Y|
+|tls-iiops|server SSL listening port|iiops|N|
+|tls-ldaps|server SSL listening port|ldaps|N|
+|tls-cbts|server listening port|CLUSTER-BROADCAST-SECURE|N|
+
+If WebLogic Administration Port is enabled on the Administration Server:
```

Comment:
If WebLogic Administration Port -> If the WebLogic administration port

Reply:
I don't see this correction: If WebLogic Administration Port -> If the WebLogic administration port